### ISSUE TITLE
Auto start for the client

### ISSUE THREAD

**Issue by osmirnov** [CityOfZion/neo-sharp#280]
Currently, NeoSharp requires user interaction to start accepting peer connections. You need to type "network start". We need to introduce another mode when the app runs and starts working without a user. It is helpful for debugging and when the app works as Windows service (Linux daemon?).

### Design
neo-sharp -i [--interactive] to run the app for a user
_or_
neo-sharp -s [--silent] to run the app in the silent mode

We can pick any approach and implement the second mode in reverse.

**Comment by shargon** [comment]
You could do like me:

1 - Create file `debug.txt`

```
network start
rpc start
```

2- Now you can call the node like this:

`neo-sharp "load debug.txt"`


3- Also, you can set your `launchSettings.json` like this:

```json
{
    "profiles":
    {
        "NeoSharp.Application":
        {
            "commandName": "Project",
            "commandLineArgs": "\"load debug.txt\""
        }
    }
}
```

**Comment by lock9** [comment]
Hello. Is this done?

**Comment by shargon** [comment]
i think that is better to pass the arguments as commands. is more flexible like now

**Comment by nunojusto** [comment]
I agree with arguments as commands, but having switches (-i and/or -s) is more easy/normal for the user.

So, we can have both.

**Comment by belane** [comment]
Agree with @nunojusto we should have both. -s for auto running network is very important for setup nodes on servers with a supervisor.

**Comment by shargon** [comment]
We should determine the behaviour of `-s`

**Comment by osmirnov** [comment]
At least, not to wait for any user interaction and to run `network start` by default.

**Comment by nunojusto** [comment]
But coding it in a way to make also possible -rpc to start RPC.
Or maybe the -s starts RPC also
